configure.ac

```diff
@@ -99,8 +95,17 @@ AC_VOMS_SOCKLEN_T
 AC_VOMS_FIND_FUNC
 AC_VOMS_STRNDUP
 
-AC_BUILD_API_ONLY
```

Comment:
Is there a reason why the call to this macro has been moved up?

Reply:
No, it's not per se necessary. I moved it up initially because I wanted it above the (original) AC_WSDL2H.
It's important to be before the GSOAP and WSDL checks of course, but I've moved it back down to just above AC_BUILD_PARTS.